01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+20.times do |x|
+  x += 1
+  str = x
+
+  if (x % 3) == 0
```

Comment:
```suggestion
  if x % 3 == 0
```

Rubyでは if の括弧を省略することが多いです。
括弧があっても問題ないのですが、省略できる時は省略してみると Rubyぽくなりますよ。